docs/en/ingest-management/getting-started-traces.asciidoc

```diff
@@ -0,0 +1,232 @@
+[[fleet-quick-start-traces]]
+[role="xpack"]
+= Quick start: Get application traces into the {stack}
+
+experimental::[]
+
+This guide describes how to:
+
+* Collect Application Performance Monitoring (APM) data
+* Send APM data to the {stack}
+* Explore and visualize the data in real-time
+
+For feedback and questions, please contact us in the {forum}[discuss forum].
+
+[discrete]
+[[fleet-prereqs-traces]]
+== Prerequisites
+
+* Please read the <<fleet-limitations,Fleet limitations>>.
+
+// To do in a future PR
```

Comment:
Looks like converts to an extra space in the HTML output.

Reply:
Ahh, because it's between bullets. OK, let's just get rid of it. I'll be fixing this week anyways.

Reply:
```suggestion

```